tests/cli/test_main_install.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (C) 2012 Anaconda, Inc
```

Comment:
These particular tests used to be in `tests/test_install.py` but since they test the `conda install` command I separated them into their own file in the `tests/cli` directory with the other CLI command-specific tests.